### PR TITLE
[201811]: security update kernel to 4.9.246

### PR DESCRIPTION
**Pull request by lguohan**
Signed-off-by: Guohan Lu <lguohan@gmail.com>

### CODE REVIEW

File: patch/preconfig/series [hunk 1]
```diff
@@ -1 +1 @@
-changelog.patch
+# changelog.patch
```
Comment (qiluo-msft) [inline patch/preconfig/series:1]: Will you remove this line and this file?

**Review by qiluo-msft — COMMENTED** [review]
As comment

File: patch/preconfig/series (hunk 1 above)
Comment (lguohan) [reply patch/preconfig/series:1]: not necessary

**Review by qiluo-msft — APPROVED** [review]

**Comment by paulmenzel** [comment]
Thank you for the update.

1.  Please split this into three commits with respective commit messages.
    a)  patch/series: Remove changelog.patch
    b)  Introduce variable for base URL
    c)  Update kernel to 4.9.246 (the tag is not needed in the commit message summary, as it’s in the correct branch)
2.  Also, it’d be great if you mentioned the security issue being fixed in the commit message of the commit updating the Linux version.
3.  Lastly, please mention if this has been tested somewhere.

**Comment by lguohan** [comment]
b) and c) is hard to split since base_url also changed.

added changelog for security issues from debian upstream

since it is taken from debian kernel, so we just did the build test.

**Comment by paulmenzel** [comment]
> b) and c) is hard to split since base_url also changed.

Next time, first introduce the variable with the “old” URL, and then change the URL.

**Pull request merged**